frontend/src/bundles/common/components/header/header.tsx

```diff
@@ -70,8 +70,12 @@ const Header: React.FC<Properties> = ({
                             pathname.match(budgetsRegex)) && (
                             <Tabs tabsData={dataTabs.dashboard} />
                         )}
-                        {pathname === `${AppRoute.WALLET}/${id}` && (
-                            <Tabs tabsData={dataTabs.wallets} />
+                        {pathname ===
+                            `${AppRoute.WALLET}/${id}${AppRoute.TRANSACTION}` && (
```

Comment:
It would be better to create a route for this component instead of this condition